akka-stream-tests/src/test/scala/akka/stream/io/TcpSpec.scala

```diff
@@ -411,12 +411,14 @@ class TcpSpec extends StreamSpec("""
         """
           akka.loglevel = DEBUG
         """).withFallback(system.settings.config))
+
       try {
         import system2.dispatcher
         val mat2 = ActorMaterializer.create(system2)
 
         val serverAddress = temporaryServerAddress()
         val binding = Tcp(system2).bindAndHandle(Flow[ByteString], serverAddress.getHostString, serverAddress.getPort)(mat2)
+        Await.ready(binding, remainingOrDefault)
```

Comment:
Use `.futureValue` for this?

Reply:
I feel wrong using futureValue without asserting on the value but i could if you prefer :)

Reply:
I think that is how we (I) have done it in this test before, no?

Reply:
/me agrees